pyproject.toml

```
@@ -30,12 +27,18 @@ exclude = [".venv/"]
 module = "tests.*"
 disallow_untyped_defs = false
 
-[tool.isort]
-profile = "black"
-
 [tool.pytest.ini_options]
 addopts = "-v --mypy -p no:warnings --cov=myproject --cov-report=html --doctest-modules --ignore=myproject/__main__.py"
 
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.ruff]
+select = ["D", "E", "F", "I"] # pydocstyle, pycodestyle, Pyflakes, isort
+
+[tool.ruff.per-file-ignores]
+"tests/*" = ["D100", "D104"]
```

```suggestion
"tests/*" = ["D100", "D104"] # Missing docstring in public module, Missing docstring in public package
```

Is it worth adding the meanings of the error codes here?